src/content/1.7/testing/_index.md

```diff
@@ -7,5 +7,99 @@ chapter: true
 
 ### Chapter 3
 
-# Testing
+# How testing works in PrestaShop
+
+> Information valid since **PrestaShop** {{< minver v="1.7.4" >}}
+
+Tests - since 1.7.4 - are located into `tests` folder. Tests are splitted into unit, integration and functional tests in PHP using PHPUnit testing framework. For end to end tests, we use Mocha.js and webdriver.io as bridge to control Selenium server.
+
+## Execute PrestaShop test suites
+
+### Execute PHPUnit test suites
+
+At least four test suites are available, testing differents parts of PrestaShop software:
+
+* `Legacy tests`: comes from PrestaShop 1.6: mix of unit, integration and functional tests;
+* `Legacy controllers`: added to help with the Symfony migration, ensures that old Back Office controllers are still runables;
+* `Admin tests`: specifics to PrestaShop 1.7, testing `src/Core` and `src/Adapter` parts of application;
+* `Symfony specific tests`: specific to PrestaShop 1.7, testing `src/PrestaShopBundle` part of application;
+
+For each of them, we need a specific configuration of PHPUnit this is why you retrieve every test suite executable as a specific composer command:
+
+* `composer phpunit-legacy`
+* `composer phpunit-controllers`
+* `composer phpunit-admin`
+* `composer phpunit-sf`
+
+> You can execute the entire PHPUnit testsuites using `composer test-all` command.
+
+### Execute Webdriver/Mocha test suite
+
+Before executing the e2e tests you need to install the dependencies and create a configuration file.
+
+1. In `tests/Selenium` folder, execute the command `npm install` (node 6+ && npm are required).
```

Comment:
It's maybe better to refer to this README: https://github.com/PrestaShop/PrestaShop/tree/develop/tests/E2E
This https://github.com/PrestaShop/PrestaShop/tree/develop/tests/Selenium are startTheme tests, there is no commits since maybe months / years

Reply:
the issue for me to do it is that for now e2e tests doesn't works on my computer : I don't want to document broken stuff

Reply:
If you tried with `tests/Selenium` instead of `tests/E2E`, that's maybe why it's broken :/

Reply:
no no, tests in tests/Selenium folder can be executed with my docs and it works. Doing what is documented in E2E folder doesn't works for me at least.

So I'm not confident to document something that may be broken.

If you're confident with it, maybe can you complete this part?

Reply:
Is it worth to copy paste it here then? look like a duplicate that will never be updated after :p

Reply:
That's why adding a little link to README files can be a solution to be sure doc doesn't contains outdated doc :) (But only when necessary)